editor/configs/routes.ts

```diff
@@ -24,7 +25,7 @@ export const routes: Route[] = [
   {
     path: "crashes",
     label: "Crashes",
-    icon: FaShieldHeart,
+    icon: FaCarBurst,
```

Comment:
improvement?

<img width="291" alt="Screenshot 2025-04-01 at 12 57 57 PM" src="https://github.com/user-attachments/assets/5c7f92b3-c89b-4410-badc-caf180b6e839" />

Reply:
I think its an improvement, especially when the sidebar is minimized

Reply:
+1 - it is easier for me to quickly tell which tab is which. 🚀